README.md

```diff
@@ -41,15 +41,33 @@ Using Dynamic Secrets means we don’t have to be concerned about them having th
 export GOPATH=$HOME/go
 export PATH=$PATH:$GOROOT/bin:$GOPATH/bin
 ```
--   Clone and go to the database plugin directory
-```sh 
-git clone https://github.com/yugabyte/hashicorp-vault-ysql-plugin && cd hashicorp-vault-ysql-plugin  
-```
--   Build the the plugin
+-   Get the plugin binary :
+
+    -   One can either Clone and Build :
```

Comment:
Generally, there is no space before `:` or `,`.
`One can either Clone and Build :` -> `One can clone and build:`